### PR TITLE
Development

### PR DESCRIPTION
**Pull request by SqlSally**

### CODE REVIEW

File: sally.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sally
```
Comment (Rugal) [inline sally.txt]: Delete and ignore files like these

File: src/Main.java [hunk 2]
```diff
@@ -2,8 +2,10 @@
 import model.Person;
```
Comment (Rugal) [inline src/Main.java:2]: It's about time to think about a good ordering of your import list

File: src/Main.java [hunk 3]
```diff
@@ -26,6 +28,7 @@ public static void main(String[] args) throws FileNotFoundException {
     String Udata;
     String Pdata;
     int num;
+    int num1;
```
Comment (Rugal) [inline src/Main.java]: Please have some meaningful variable, variable name like this doesn't make sense to anyone

File: src/Main.java [hunk 4]
```diff
@@ -61,9 +64,47 @@ public static void main(String[] args) throws FileNotFoundException {
     do {
       num = scanner.nextInt();
       screen.selection(num);
+      break;
+
     } while (num != 0);
 
-    System.out.println("You have log out");
+//    System.out.println("You have log out");
```
Comment (Rugal) [inline src/Main.java]: delete comment

File: src/model/WritingToFiles.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
```
Comment (Rugal) [inline src/model/WritingToFiles.java]: Remove license header

File: src/model/WritingToFiles.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    File file = new File("people");
+    FileWriter fw = new FileWriter(file,true);
+    PrintWriter pw = new PrintWriter(fw);
+    
+    pw.println( "\n student" + " " + username + " "+ password );
+    
+    pw.close();
+    
+  }
+
```
Comment (Rugal) [inline src/model/WritingToFiles.java]: Delete blank lines

File: src/model/WritingToFiles.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+import java.io.IOException;
+import java.io.PrintWriter;
+
+/**
+ *
+ * @author sally
+ */
+public class WritingToFiles {
+  
+  public static void addStudent(String username, String password) throws IOException{
```
Comment (Rugal) [inline src/model/WritingToFiles.java]: Maybe you should start thinking in OOP way

File: src/view/AdminScreen.java [hunk 8]
```diff
@@ -1,5 +1,11 @@
 package view;
 
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import static jdk.nashorn.internal.objects.NativeRegExp.test;
```
Comment (Rugal) [inline src/view/AdminScreen.java]: Import ordering

File: src/view/AdminScreen.java [hunk 9]
```diff
@@ -39,5 +45,5 @@ public void selection(int num) {
         break;
     }
   }
-
+
```
Comment (Rugal) [inline src/view/AdminScreen.java]: Why having newlines?

File: test.txt [hunk 10]
```diff
@@ -0,0 +1,3 @@
+line 1
```
Comment (Rugal) [inline test.txt]: What's this file?

**Review by Rugal — CHANGES_REQUESTED** [review]

File: test.txt (hunk 10 above)
Comment (SqlSally) [reply test.txt]: delete it

File: src/view/AdminScreen.java (hunk 8 above)
Comment (SqlSally) [reply src/view/AdminScreen.java]: delete it

**Review by Rugal — APPROVED** [review]

**Pull request merged**